javascript/classes.json

```diff
@@ -490,6 +490,7 @@
             },
             "firefox": {
```

Comment:
We ought to replace what we have here (and in Firefox for Android) with something like:

```
{
  "version_added": false
}
```

It's a bit of a quirky situation though, so we could plausibly provide a "See <a href='bugzil.la/1562054'>bug 1562054</a>." note here, if you like.

Reply:
is it okay to add the version information about which nightly supports the feature, in "Notes" section?
or should we avoid that?

Reply:
I think it's best to avoid it, at least in this case. There's very little expectation that developers are going to run old nightly releases and the version numbers can confuse things for stable users.